maxim/utils/utils.py

```
@@ -29,8 +30,8 @@ def generate_cuid() -> str:
         # Fingerprint
         try:
             hostname = socket.gethostname()
-        except:
-            hostname = 'unknown'
+        except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)